source/boot_security.c

```diff
@@ -1,11 +1,37 @@
 #include "boot_security.h"
+#include "simulation.h"
+
+/* Remove the include */
+#include <windows.h> // only runs on Windows
+
+#include "mbedtls\aes.h"
+#include <stdio.h>
+#include <string.h>
+#include <stdlib.h>
```

Comment:
Please remove the standards library includes between the line 8 to 10. All they come from "boot_types.h."